### PR TITLE
k8s 1.14.3 from cloudconfig

### PR DESCRIPTION
**Pull request by MarcelMue**
These are changes which are necessary to make clusters with k8s 1.14.3 work.

### CODE REVIEW

File: service/controller/clusterapi/v28/cloudconfig/cloud_config.go [hunk 1]
```diff
@@ -10,9 +10,9 @@ import (
 )
 
 const (
-	FileOwnerUser  = "root"
-	FileOwnerGroup = "root"
-	FilePermission = 0700
+	FileOwnerUserName  = "root"
```
Comment (MarcelMue) [inline service/controller/clusterapi/v28/cloudconfig/cloud_config.go]: I did my best to follow the changes introduced to `azure-operator` here https://github.com/giantswarm/azure-operator/pull/519

I don't know much about file permissions / importance here though, so I mainly just made the operator compile again.

File: service/controller/clusterapi/v28/cloudconfig/master_template.go [hunk 2]
```diff
@@ -127,17 +127,25 @@ func (e *MasterExtension) Files() ([]k8scloudconfig.FileAsset, error) {
 			AssetContent: cloudconfig.DecryptTLSAssetsScript,
 			Path:         "/opt/bin/decrypt-tls-assets",
 			Owner: k8scloudconfig.Owner{
-				User:  FileOwnerUser,
-				Group: FileOwnerGroup,
+				Group: k8scloudconfig.Group{
```
Comment (MarcelMue) [inline service/controller/clusterapi/v28/cloudconfig/master_template.go]: All of these changes came from the change of file permission setup in k8scc.

File: service/controller/legacy/v28/key/key.go [hunk 3]
```diff
@@ -700,30 +700,30 @@ func ImageID(customObject v1alpha1.AWSConfig) (string, error) {
...
-		"eu-west-2":      "ami-069966bea0809e21d",
-		"eu-west-3":      "ami-0194c504244182155",
-		"sa-east-1":      "ami-0cd830cc037613a7d",
-		"us-east-1":      "ami-08e58b93705fb503f",
-		"us-east-2":      "ami-03172282aaa2899be",
-		"us-gov-east-1":  "ami-0ff9e298ea0bacf53",
-		"us-gov-west-1":  "ami-e7f59e86",
-		"us-west-1":      "ami-08d3e245ebf4d560f",
-		"us-west-2":      "ami-0a4f49b2488e15346",
+		"ap-northeast-1": "ami-036857bdeb5b3362c",
```
Comment (MarcelMue) [inline service/controller/legacy/v28/key/key.go]: The usual game of copy / pasting the correct amis.

Comment (xh3b4sd) [reply service/controller/legacy/v28/key/key.go]: We need to maintain this for the `clusterapi` controllers as well. See https://github.com/giantswarm/aws-operator/blob/master/service/controller/clusterapi/v28/key/cluster.go#L348-L378.

Comment (MarcelMue) [reply service/controller/legacy/v28/key/key.go]: I aligned this now - thanks for the eagle eyes!

PTAL

**Comment by mazzy89** [comment]
Going to solve conflict here

**Comment by mazzy89** [comment]
@MarcelMue last check before merging 🙏

File: service/controller/legacy/v28/version_bundle.go [hunk 4]
```diff
@@ -8,31 +8,46 @@ func VersionBundle() versionbundle.Bundle {
...
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "calico",
+				Description: "Update calico to 3.7.2. More info here: https://docs.projectcalico.org/v3.7/release-notes/",
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "containerlinux",
+				Description: "Updated to 2079.5.1. More info here: https://github.com/coreos/manifest/releases/tag/v2079.5.1",
```
Comment (xh3b4sd) [inline service/controller/legacy/v28/version_bundle.go]: Tense needs to be imperative here and below. When you review your own changes you can spot most of these issues.

Comment (mazzy89) [reply service/controller/legacy/v28/version_bundle.go]: Right. Forgot it here.

**Comment by mazzy89** [comment]
@MarcelMue @xh3b4sd ready here?

**Comment by MarcelMue** [comment]
I think so.

**Comment by mazzy89** [comment]
Merging

**Pull request merged**